griptape/labs/tools/aws_redshift/tool.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import time
+from typing import Optional
+from attr import define, field
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.core import BaseTool
+from griptape.core.decorators import activity
+from schema import Schema, Literal
+import boto3
+
+
+@define
+class AwsRedshiftClient(BaseTool):
+    database: str = field(default=None, kw_only=True)
+
+    workgroup_name: str = field(default=None, kw_only=True)
+
+    aws_region_name = field(default=None, kw_only=True)
```

Comment:
Add a type hint and default to something. May be "us-east-1" or `None`. If `None` then don't set it in boto3.